src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -612,20 +612,71 @@ cdef class WidomInsertion(ReactionAlgorithm):
 
         self._set_params_in_es_core()
 
-    def measure_excess_chemical_potential(self, reaction_id=0):
+    def calculate_particle_insertion_potential_energy(self, reaction_id=0):
         """
-        Measures the excess chemical potential in a homogeneous system for
-        the provided ``reaction_id``. Please define the insertion moves
-        first by calling the method :meth:`~ReactionAlgorithm.add_reaction`
-        (with only product types specified).
-        Returns the excess chemical potential and the standard error for
-        the excess chemical potential. The error estimate assumes that
-        your samples are uncorrelated.
-
+        Measures the potential energy when particles are inserted in the system
+        following the reaction  provided ``reaction_id``. Please define the 
+        insertion moves first by calling the method 
+        :meth:`~ReactionAlgorithm.add_reaction` (with only product types specified).
+        Note that altought this function does not provide directly the chemical potential,
+        it can be used to calculate it. For an example of such application please check 
+        the example script in /samples/widom_insertion.py
         """
         # make inverse widom scheme (deletion of particles) inaccessible.
         # The deletion reactions are the odd reaction_ids
         if(reaction_id < 0 or reaction_id > (deref(self.WidomInsertionPtr).reactions.size() + 1) / 2):
             raise ValueError("This reaction is not present")
-        return deref(self.WidomInsertionPtr).measure_excess_chemical_potential(
+        return deref(self.WidomInsertionPtr).calculate_particle_insertion_potential_energy(
             deref(self.WidomInsertionPtr).reactions[int(2 * reaction_id)])
+
+    def calculate_excess_chemical_potential(
+            self, particle_insertion_potential_energy_samples, N_blocks=16):
+        """
+        Given a set of samples of the particle insertion potential energy,
+        calculates the excess chemical potential and its statistical error.
+        Parameters
+        ----------
+        particle_insertion_potential_energy_samples : array_like of :obj:`float`
+            Samples of the particle insertion potential energy.
+        N_blocks : :obj:`int`, optional
+            Number of bins for binning analysis.
+        Returns
+        -------
+        mean : :obj:`float`
+            Mean excess chemical potential.
+        error : :obj:`float`
+            Standard deviation of the mean.
```

Comment:
I don't think that's true. If I'm reading the block analysis code correctly, it currently returns the standard error of the mean, not the standard deviation.